app/services/wiki_service.rb

```diff
@@ -0,0 +1,35 @@
+class WikiService
+  include HTTParty
+  attr_reader :base_uri, :default_params
+
+  def initialize
+    @base_uri = "http://en.wikipedia.org/w/api.php"
+    @default_params = { action: "query",
+                        prop: "revisions",
+                        format: "json",
+                        rvprop: "content",
+                      }
+  end
+
+  def collect_data(titles)
+    default_params[:titles] = titles
+    summaries(responses).map { |summary| summary.split("\n")
+                                                .first.gsub(/[\]}=\/{|*+\\\[<>]/, "") }
```

Comment:
Line is too long. [87/80]<br>Expression at 17, 87 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 44) spaces for indenting an expression spanning multiple lines.